cloud-config/master.yaml.tmpl

```diff
@@ -784,7 +784,7 @@ write_files:
       variables-hash-bucket-size: "128"
       server-name-hash-bucket-size: "1024"
       server-name-hash-max-size: "1024"
-- path: /srv/ingress-controller-sa.yaml
+- path: /srv/ingress-controller-sa.yml
```

Comment:
I would rather make the changes towards `yaml` as we use that for most of our files in and in  other projects too.

Reply:
Makes sense. See new changes